src/models/common/star.rs

```diff
@@ -0,0 +1,42 @@
+use crate::*;
+
+use rsb_derive::Builder;
+use rvstruct::ValueStruct;
+use serde::{Deserialize, Serialize};
+use serde_with::skip_serializing_none;
+
+#[derive(Debug, PartialEq, Clone, Serialize, Deserialize)]
+#[serde(tag = "type")]
+#[serde(rename_all = "lowercase")]
+#[allow(clippy::large_enum_variant)]
+pub enum SlackStarsItem {
+    Message {
+        message: SlackHistoryMessage,
```

Comment:
Would be great to extract those into separate structures for consistency with other types for other API structures.

Reply:
I have updated the PR to use separate structures 👍